packages/gappa/gappa.1.4.2/opam

```diff
@@ -0,0 +1,46 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+authors: "Guillaume Melquiond"
+bug-reports: "https://gitlab.inria.fr/gappa/gappa/-/issues"
+homepage: "https://gitlab.inria.fr/gappa/gappa"
+dev-repo: "git+https://gitlab.inria.fr/gappa/gappa.git"
+license: "CECILL-2.1"
+patches: [
+  "0001-Added-configure-for-c-11.patch"
```

Comment:
I guess so. I was not aware of this mechanism. I am the original author of this patch and used to copy it from version to version. Can you point me to the documentation of this "auto extract patches from commits and put it somewhere else" mechanism, so that I know, what I am doing?